analytics/criteo.js

```diff
@@ -0,0 +1,109 @@
+//
+// Criteo tracking for auctions product feed and artworks product feed.
+//
+
+window.criteo_q = window.criteo_q || []
+var pathSplit = location.pathname.split("/")
+var userEmail = function() {
+  return sd.CURRENT_USER ? [sd.CURRENT_USER.email] : [];
+}()
+if (pathSplit[1] === "auctions") {
+  // Auctions events
+  window.criteo_q.push(
+    { event: "setAccount", account: sd.CRITEO_AUCTIONS_ACCOUNT_NUMBER },
+    { event: "setSiteType", type: "m" },
+    { event: "viewHome" }
+  )
+} else if (pathSplit[1] === "auction") {
```

Comment:
Ah, I'm seeing now that you have a nested conditional. To remove duplication and make this a little more clear you could probably check for the auction string and the nested `!pathSplit[3]` check in one line like so `else if (pathSplit[1] === 'auction' && !pathSplit[3])`.